README.md

```diff
@@ -281,6 +282,24 @@ sources:
 
 </details>
 
+#### Block Tracking
+
+The Graph Client can track block numbers and do the following queries by following [this pattern](https://thegraph.com/docs/en/developer/distributed-systems/#polling-for-updated-data) with `blockTracking` transform;
+
+```yaml
+sources:
+  - name: uniswapv2
+    handler:
+      graphql:
+        endpoint: https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2
+    transforms:
+      - blockTracking:
```

Comment:
@ardatan since this is applied on the source level, maybe we can add a list of operationNames to ignore? 🤔

Reply:
Yes! It is implemented now!